portal/server/helpers/map-facility-properties.ts

```diff
@@ -0,0 +1,38 @@
+import { FacilityDashboard, FACILITY_STAGE } from '@ukef/dtfs2-common';
+
+/**
+ * Maps the facility stage based on the current facility properties.
+ *
+ * @param {Facility} facility - The facility object to map the stage for.
+ * @returns {Facility} The facility object with the mapped stage.
```

Comment:
This contradicts the other type & its also not importing correctly
```suggestion
 * @param facility - The facility object to map the stage for.
 * @returnsThe facility object with the mapped stage.
```

Reply:
Have updated the docstrings.

Reply:
It's probably safer to not include any type in the jsdoc so that we have one source of truth & don't override the typescript but will resolve anyway